doc/api/https.md

```diff
@@ -10,11 +10,29 @@ separate module.
 ## Class: https.Agent
 <!-- YAML
 added: v0.4.5
+changes:
+  - version: v2.5.0
+    pr-url: https://github.com/nodejs/node/pull/2228
+    description: parameter `maxCachedSessions` added to `options` for TLS
+                 sessions reuse.
+  - version: v5.3.0
+    pr-url: https://github.com/nodejs/node/pull/4252
+    description: support `0` `maxCachedSessions` to disable TLS session caching.
 -->
 
 An [`Agent`][] object for HTTPS similar to [`http.Agent`][]. See
 [`https.request()`][] for more information.
 
+### new Agent([options])
+
+* `options` {Object} Set of configurable options to set on the agent.
+  Can have the same fields as for [`http.Agent(options)`][], and
+  * `maxCachedSessions` {number} maximum number of TLS cached sessions.
+    Use `0` to disable TLS session caching.
+    **Default:** `100`.
+
+    See [`tls SessionResumption`][] for infomation about TLS session reuse.
```

Comment:
Nit: I would use just "Session Resumption" for the link name, that's the actual chapter title on TLS docs.